packages/core/content-releases/server/src/types.ts

```diff
@@ -1,14 +1,16 @@
+import type { Entity, Common } from '@strapi/types';
+
 // @TODO: Probably user & role types should be imported from a common package
```

Comment:
are these types shared between the server and the admin?

Reply:
If they are I would say this file would be better placed inside content-releases/shared

Reply:
That sounds like a good idea! I just didn't consider it 🙃

Reply:
We have a shared folder with contracts for the controllers too !https://github.com/strapi/strapi/blob/ts/admin/packages/core/admin/shared/contracts/admin.ts

It can be handy specially if the front and backend work is splitted between multiple devs.